### PR TITLE
Add Obihai Integration

### PR DESCRIPTION
**Pull request by dshokouhi**
**Description:**

Docs for Obihai integration

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26537

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by dshokouhi** [comment]
hmm I am not sure why this is failing, this is my first time creating a page for the docs did I make a mistake somewhere?

**Comment by dshokouhi** [comment]
that merge may not be good lol chances of closing this PR for a new one are very high

File: source/_components/obihai.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Obihai"
+description: "Instructions on how to integrate your Obihai device into Home Assistant."
+logo: obitalk.png
+ha_iot_class: "Local Polling"
+ha_category:
+  - Sensor
+ha_release: 0.99
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/obihai.markdown]: Please remove the `redirect_from`

File: source/_components/obihai.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+The `obihai` integration allows you to view the call status for your [Obihai devices](https://www.obitalk.com/info/products#home_section).
+
+To enable `obihai` in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: obihai
```
Comment (klaasnicolaas) [inline source/_components/obihai.markdown:18]: Only required items in the first example

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/obihai.markdown (hunk 1 above)
Comment (dshokouhi) [reply source/_components/obihai.markdown]: removed

File: source/_components/obihai.markdown (hunk 2 above)
Comment (dshokouhi) [reply source/_components/obihai.markdown:18]: done

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**